filesystem/src/test/java/org/frankframework/filesystem/smb/Samba2FileSystemTest.java

```diff
@@ -2,18 +2,24 @@
 
 import static org.junit.jupiter.api.Assumptions.assumeFalse;
 
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 import org.frankframework.filesystem.FileSystemTest;
 import org.frankframework.filesystem.IFileSystemTestHelper;
 import org.frankframework.filesystem.LocalFileSystemTestHelper;
-import org.frankframework.filesystem.smb.Samba2FileSystem.Samba2AuthType;
 import org.frankframework.testutil.junit.LocalFileServer;
-import org.frankframework.testutil.junit.LocalFileServer.FileSystemType;
 import org.frankframework.testutil.junit.LocalFileSystemMock;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
 
+/**
+ * Test for the Samba2FileSystem. Without changes, it runs with a stubbed Samba implementation.
```

Comment:
Het is geen stub, het is een 'echte' server gemaakt in Java.

Reply:
Aangepast.